### PR TITLE
chore: fix changelog links 

### PR DESCRIPTION
**Pull request by jmalicevic**
---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments
- [ ] Title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

### CODE REVIEW

File: .changelog/unreleased/features/3622-e2e-mempool-lanes.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 - `[e2e]` Added support for mempool lanes in e2e. 
-  ([#3622](https://github.com/tendermint/tendermint/pull/3622))
+  ([#3622](https://github.com/cometbft/cometbft/pull/3622))
```
Comment (hvanz) [inline .changelog/unreleased/features/3622-e2e-mempool-lanes.md:2]:
```suggestion
  ([#3503](https://github.com/cometbft/cometbft/issues/3503))
```

File: .changelog/unreleased/features/3622-kvstore-mempool-lanes.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 - `[kvstore]` Extended `CheckTx` in kvstoreApp to support mempool lanes. 
-  ([#3622](https://github.com/tendermint/tendermint/pull/3622))
+  ([#3622](https://github.com/cometbft/cometbft/pull/3622))
```
Comment (hvanz) [inline .changelog/unreleased/features/3622-kvstore-mempool-lanes.md:2]:
```suggestion
  ([#3503](https://github.com/cometbft/cometbft/issues/3503))
```

**Review by hvanz — APPROVED** [review]
👍

File: .changelog/unreleased/features/3622-kvstore-mempool-lanes.md (hunk 2 above)
Comment (jmalicevic) [reply .changelog/unreleased/features/3622-kvstore-mempool-lanes.md:2]: According to our contributing guidelines, PRs are preferred.

**Pull request merged**